### ISSUE TITLE
Set reasonable fallback values for the `classNameGenerator` and `inlineStyleGenerator` options

### ISSUE THREAD

**Issue by ncoetzer** [10up/block-components#206]
### Describe the bug

Using the `registerBlockExtension` API without passing in the `classNameGenerator` and `inlineStyleGenerator` functions/options results in an error in the admin.

This would be helpful if someone wanted to use the API to extend the block's attributes and handle any logic within the markup. For example, extending the core/button with a `TextareaControl` in the InspectorControls to specify text that replaces what screenreader will read as the link text.

### Steps to Reproduce

1. When registering the extension using the function without specifying the class and inline style functions
```
registerBlockExtension(blockName, {
	extensionName: 'links-with-accessible-text',
	attributes: additionalAttributes,
	Edit: BlockEdit,
});
```
### Screenshots, screen recording, code snippet

<img width="839" alt="image" src="https://user-images.githubusercontent.com/3490023/222164894-b2a97771-b628-4156-930f-38caca108f71.png">
<img width="410" alt="image" src="https://user-images.githubusercontent.com/3490023/222165055-7494fecc-ab5a-4d24-947e-0f210e1987ef.png">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fabiankaegy** [comment]
@ncoetzer Does this error also happen when you just have one of the two defined? Meaning either `classNameGenerator` **OR** `inlineStyleGenerator`?

**Comment by ncoetzer** [comment]
@fabiankaegy Seems like it only happens if `classNameGenerator` is not passed to the function. It works fine with **OR** without `inlineStyleGenerator`.

**Comment by fabiankaegy** [comment]
Okay yeah then this is a remnant of me adding the `inlineStyleGenerator` in #197 but not handling the case where no `classNameGenerator` is defined. Thanks for catching it :)

**Comment by ncoetzer** [comment]
@fabiankaegy Oh, I see what you mean. We could probably add the same conditional check as on line [127](https://github.com/10up/block-components/pull/197/files#diff-214abe1434a74fac9071fb3cf7421435dc469c093bba3ecaeb91fdc5655d6eb8R127) for the `classNameGenerator`?

**Comment by fabiankaegy** [comment]
@ncoetzer yeah probably :)

**Comment by ncoetzer** [comment]
@fabiankaegy Thanks! I have this on my list along with the Image task from before 🙂

**Comment by fabiankaegy** [comment]
This was fixed in #199
